src/workarounds/cuda_arrays.jl

```diff
@@ -1,16 +1,5 @@
 using LinearAlgebra
 
-# https://github.com/JuliaGPU/CUDA.jl/issues/1572
-function LinearAlgebra.eigen(A::Hermitian{<:Complex,<:CUDA.CuArray})
-    values, vectors = CUDA.CUSOLVER.heevd!('V', 'U', A.data)
-    (; values, vectors)
-end
-
-function LinearAlgebra.eigen(A::Hermitian{<:Real,<:CUDA.CuArray})
-    values, vectors = CUDA.CUSOLVER.syevd!('V', 'U', A.data)
-    (; values, vectors)
-end
-
```

Comment:
Are you sure this works? The issue is still open.

Reply:
> Also while you are at it, could you remove the workarounds I introduced in

> https://github.com/JuliaMolSim/DFTK.jl/commit/6a8c25c22999d6153b98574ebd99211779c8dbae#diff-21d2f263953fee8479af0575d1c861d2659db2fb962f8a3d94c26dc263004223

I thought you meant to remove it? Seems to work for my local tests. I think the methods are now in CUDA.jl via this PR: https://github.com/JuliaGPU/CUDA.jl/pull/1603/files#diff-e1fa24914853993ba813e2b2ee908a06f18ebcf7282c9dc9fc3e47842e91aac5?

Reply:
Indeed, your're right this now works. Great to see that :+1:.

But the workarounds I was talking about was in the test/gpu.jl file.